python/mxnet/gluon/trainer.py

```diff
@@ -43,6 +43,12 @@ class Trainer(object):
     kvstore : str or KVStore
         kvstore type for multi-gpu and distributed training. See help on
         :any:`mxnet.kvstore.create` for more information.
+
+    Properties
+    ----------
+    Learning_rate: float
+        The learning rate of the optimizer or the learning rate of the
+        LRScheduler of the optimizer if the LRScheduler is defined.
```

Comment:
explain it can be accessed with trainer.learning_rate and set with trainer.learning_rate = xxx

Reply:
resolved